pages/accountLists/[accountListId]/settings/notifications.page.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { notificationSettingsMocks } from 'src/components/Settings/notifications/notificationSettingsMocks';
+import theme from 'src/theme';
+import Notifications from './notifications.page';
+
+const accountListId = 'account-list-1';
+
+const mockEnqueue = jest.fn();
+const mutationSpy = jest.fn();
+const push = jest.fn();
+
+const router = {
+  query: { accountListId },
+  isReady: true,
+  push,
+};
+
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+const MocksProviders = (props: { children: JSX.Element; setup?: string }) => (
+  <ThemeProvider theme={theme}>
+    <TestRouter router={router}>
+      <GqlMockedProvider
+        mocks={{
+          notificationSettingsMocks,
```

Comment:
Unless you spread this, you're mocking an operation named `notificationSettingsMocks`, which doesn't exist.

```suggestion
          { ...notificationSettingsMocks },
```